decks/emoji.mdx

```diff
@@ -0,0 +1,98 @@
+import GreetingLoader from "./src/greeting-loader";
+ 
+export const theme = customTheme;
+ 
+# Workshop Proposal: Emoji Sentiment Data
+
+ 
+---
+
+# Specific Idea
+Ideas:
+1. predicting a tweet's emotion based on emojis used
+- predicting a tweet’s emotion -> generalize to predict general attitude from multiple tweets on a specific event? (tracking hashtags) / predict someone’s general feeling (in a week/month/year) by analyzing emojis in a person’s tweets.
+- what tweets are we analyzing?
+2. words to emojis: emotion translator (wrote about this below!)
+ 
+---
+ 
+# Interactive Component
+- walk thru concepts taught and teach to translate concepts into code
+- present prompts and ask workshop participants how they would code it
+- then search for and use a recent tweet from a celebrity or workshop participant and test code
+- or search for and use any form of text from the internet and ask all participants to test on text
+ 
+---
+ 
+# Code Deliverables
+- week-by-week plan:
+ - week 1: code for reading and analyzing tweets/ texts
+ - week 2: code for identifying key terms for the emotions they convey (idea #2) / identifying the emojis in tweets (idea #1)
+ - week 3: code for incorporating tech for translating between emotions and emojis
+ 
+---
+ 
+# Slide Outline
+- PART 1: Introduction and motivation
+ - introduce workshop and motivation behind it
+ - define emojis and their relation to emotion and communication
+ - goal of the workshop, what coding techniques and technologies are being taught
+- PART 2: Concepts and code breakdown
+ - introduce concepts (analyzing tweets? Dataset, unicode value?), new coding techniques and terms (twitter API, python libraries)
+ - display how concepts translate into code concepts
+ - walkthrough of coding concepts, use graphs and visuals
+- PART 3: Interactive activity
+ - have participants try writing code on their own
+ - suggest an action program should perform and help participants write code for it (activity short examples)
+ - for example, code from translating "happy" into smiley face emoji (ideas #2)
+- for example, code for recognizing a smiley face and output by labeling it as a positive tweets (idea #1)
+ 
+---
+ 
+# Timing
+- time limit: 90 min to 120 min
+- PART 1: 15 min
+- PART 2: 45 min
+- PART 3: 30 min
+ 
+---
+ 
+# Motivation Behind Projects
+- impact and influence of emojis in communication is increasing
+- emojis have been taking over the way people communicate with each other online
+- emojis help by adding personality and clarity with conveyed emotions
+- sometimes people express their attitudes and opinions vaguely through words, analyzing emojis could help predict the tweet’s attitude towards certain events. (- can apply to social, political… issues)
+- workshop can show how people interpret text into emotion
+ 
+---
+ 
+# Difficulty/Prior experience
+- easy/medium
+- basic python knowledge (ECS32A level)
+- prior experience with twitter API? 
+ 

```

Comment:
readers might need to set up twitter developer credentials to use API. find out how to do this asap